generated/typespec-ts/src/models/models.ts

```diff
@@ -0,0 +1,914 @@
+   */
+  validationFile?: string | null;
+  /**
+   * The name of the model to fine-tune. You can select one of the
+   * [supported models](/docs/guides/fine-tuning/what-models-can-be-fine-tuned).
+   */
+  model: string | "babbage-002" | "davinci-002" | "gpt-3.5-turbo";
+  /** The hyperparameters used for the fine-tuning job. */
+  hyperparameters?;
+  /**
+   * A string of up to 18 characters that will be added to your fine-tuned model name.
+   *
+   * For example, a `suffix` of "custom-model-name" would produce a model name like
+   * `ft:gpt-3.5-turbo:openai:custom-model-name:7p4lURel`.
+   */
+  suffix?: string | null;
+}
+
+export interface FineTuningJob {
+  /** The object identifier, which can be referenced in the API endpoints. */
+  id: string;
+  /** The object type, which is always "fine_tuning.job". */
+  object: "fine_tuning.job";
+  /** The Unix timestamp (in seconds) for when the fine-tuning job was created. */
+  createdAt: Date;
+  /**
+   * The Unix timestamp (in seconds) for when the fine-tuning job was finished. The value will be
+   * null if the fine-tuning job is still running.
+   */
+  finishedAt: Date | null;
+  /** The base model that is being fine-tuned. */
+  model: string;
+  /**
+   * The name of the fine-tuned model that is being created. The value will be null if the
+   * fine-tuning job is still running.
+   */
+  fineTunedModel: string | null;
+  /** The organization that owns the fine-tuning job. */
+  organizationId: string;
+  /**
+   * The current status of the fine-tuning job, which can be either `created`, `pending`, `running`,
+   * `succeeded`, `failed`, or `cancelled`.
+   */
+  status:
+    | "created"
+    | "pending"
+    | "running"
+    | "succeeded"
+    | "failed"
+    | "cancelled";
+  /**
+   * The hyperparameters used for the fine-tuning job. See the
+   * [fine-tuning guide](/docs/guides/fine-tuning) for more details.
+   */
+  hyperparameters;
+  /**
+   * The file ID used for training. You can retrieve the training data with the
+   * [Files API](/docs/api-reference/files/retrieve-contents).
+   */
+  trainingFile: string;
+  /**
+   * The file ID used for validation. You can retrieve the validation results with the
+   * [Files API](/docs/api-reference/files/retrieve-contents).
+   */
+  validationFile: string | null;
+  /**
+   * The compiled results file ID(s) for the fine-tuning job. You can retrieve the results with the
+   * [Files API](/docs/api-reference/files/retrieve-contents).
+   */
+  resultFiles: string[];
+  /**
+   * The total number of billable tokens processed by this fine tuning job. The value will be null
+   * if the fine-tuning job is still running.
+   */
+  trainedTokens: number | null;
+  /**
+   * For fine-tuning jobs that have `failed`, this will contain more information on the cause of the
+   * failure.
+   */
+  error: null;
+}
+
+export interface ListPaginatedFineTuningJobsResponse {
+  object: string;
+  data: FineTuningJob[];
```

Comment:
[Compile Error]

  src/models/models.ts:357:3
    357   data: FineTuningJob[];
          ~~~~
    The expected type comes from property 'data' which is declared here on type 'ListPaginatedFineTuningJobsResponse